pyproject.toml

```diff
@@ -1,3 +1,10 @@
 [tool.black]
 line-length = 100
 target-version = ['py38', 'py39']
+
+[tool.isort]
+profile = "black"
+py_version = 38
+line_length = 100
```

Comment:
```suggestion
line_length = 100
combine_as_imports = true
```
I'd suggest using `combine_as_imports`, which avoids changes like this:
![image](https://user-images.githubusercontent.com/8530778/143679546-7a57cd24-beb2-45c4-ae68-a19c2c6b42c7.png)

Reply:
Oh yeah, looks better, thank you, I'm going to apply this new configuration later.